### DISCUSSION TITLE
I hacked the build script to dump .diffs

### DISCUSSION THREAD

**Discussion by rasa** [okibcn/nano-for-windows#2] [Show and tell]
See
https://github.com/rasa/nano-for-windows/actions/runs/4030103674/jobs/6928545389#step:7:187
I could submit this as a PR, if you want.
Best,
Ross

**Comment by okibcn** [comment]
Thanks, @rasa. When I started, I was considering creating patches after the modification and applying them, but that would be extra work, and it would require the maintenance of the patch files.   This way everything is integrated into a shell script. 

However, your b() function provides valuable info for debugging in case something fails directly in the action log. I will integrate it, it is a good idea!!!  I will change the order of the diff printout to make them easier to read. I also have to add one more patch that was released in build 10163.

Hey, thanks for the improvement.
